pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2823,18 +2827,25 @@ func maybeAssignableRem(ctx *domainContext) {
 	for _, long := range assignments {
 		err := pciAssignableRemove(long)
 		if err != nil {
-			log.Errorf("updateUsbAccess remove failed: %s\n", err)
+			log.Errorf("maybeAssignableRem remove failed: %s\n", err)
 		}
 	}
 	if len(assignments) != 0 {
 		ctx.publishAssignableAdapters()
 	}
 }
 
-func handleIBDelete(ctx *domainContext, ib types.IoBundle,
-	status *types.AssignableAdapters) {
+func handleIBDelete(ctx *domainContext, name string) {
+
+	log.Infof("handleIBDelete(%s)", name)
+	aa := ctx.assignableAdapters
+
+	ib := aa.LookupIoBundle(0, name)
```

Comment:
We really should remove the type check completely!

Reply:
Opened up a story for this..Will take care of it as a separate merge.